django/views/debug.py

```diff
@@ -507,6 +516,7 @@ def technical_404_response(request, exception):
         'urlpatterns': tried,
         'reason': str(exception),
         'request': request,
+        'META': get_safe_request_meta(request),
```

Comment:
Took a look at this, technical_404_response doesn't print request.META attributes besides the request.META.REQUEST_METHOD attr, so I will remove the above line.